src/auth/permissions/index.ts

```diff
@@ -148,7 +148,11 @@ const Permissions: PermissionInterface = {
         'access_role_permission',
         'permission.permission_id',
         'access_role_permission.permission_id')
-      .select()
+      .select({
```

Comment:
How come you've removed `access_role_id`?

Reply:
It's not useful. We should deal with the value it describes instead.